jenkins-scripts/docker/lib/docker_generate_dockerfile.bash

```diff
@@ -417,12 +411,30 @@ echo '# END SECTION'
 BUILDSH_CCACHE
 fi
 
+cat >> Dockerfile << DELIM_DOCKER_USER
+# Create a user with passwordless sudo
+ARG USERID
+ARG USER
+RUN adduser --uid \$USERID --gecos "Developer" --disabled-password \$USER
```

Comment:
I would recommend a slightly more complete arguments to the user. In particular set the group ID as well. 

I have a good example at: https://github.com/osrf/rocker/blob/acbbff59d6aef6a9654eabc0ff7e3f823936d6e3/src/rocker/templates/user_snippet.Dockerfile.em#L9

Reply:
Thanks! I've included GID handlng in  ef59662